From 9f28f973b2e744289744c23b84ddd297a88c25a7 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 9 Feb 2020 00:48:21 +0200
Subject: [PATCH 092/186] Fix enter deep doze when moving

  * from https://github.com/LineageOS/android_frameworks_base/commit/0486877683147d4c311574c2123c9cb7d2dfadaa

	modified:   services/core/java/com/android/server/AnyMotionDetector.java
---
 services/core/java/com/android/server/AnyMotionDetector.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/AnyMotionDetector.java b/services/core/java/com/android/server/AnyMotionDetector.java
index 8c5ee7f3502..316306df4f4 100644
--- a/services/core/java/com/android/server/AnyMotionDetector.java
+++ b/services/core/java/com/android/server/AnyMotionDetector.java
@@ -231,8 +231,8 @@ public class AnyMotionDetector {
                 Slog.d(TAG, "mCurrentGravityVector = " + currentGravityVectorString);
                 Slog.d(TAG, "mPreviousGravityVector = " + previousGravityVectorString);
             }
-            mRunningStats.reset();
             status = getStationaryStatus();
+            mRunningStats.reset();
             if (DEBUG) Slog.d(TAG, "getStationaryStatus() returned " + status);
             if (status != RESULT_UNKNOWN) {
                 if (mWakeLock.isHeld()) {
-- 
2.17.1

